sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/RequiresStableInputIT.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why is this an IT within `google-cloud-platform` instead of within `sdks/java/core`?

`@RequiresStableInput` is an SDK property that isn't limited to GCP.

Reply:
Thanks for your comments, Luke! The other comments look good to me and I am iterating my code based on them.
For this comment, I put the test here now because this is the project root for "googleCloudPlatformIntegrationTest". The other choice is "examplesJavaIntegrationTest", which is not where it should go. I could create a third type of postCommit task, like "JavaSDKIntegrationTest". What do you think?